Rubberduck.Parsing/VBA/COMReferenceSynchronizerBase.cs

```diff
@@ -80,12 +68,14 @@ public void SyncComReferences(IReadOnlyList<IVBProject> projects, CancellationTo
                 unmapped.Add(item);
             }
 
-            if (unmapped.Any())
+            if (!unmapped.Any())
             {
-                foreach (var reference in unmapped)
-                {
-                    UnloadComReference(reference, projects);
-                }
+                return;
+            }
```

Comment:
instead of inverting that if-condition, it can be completely removed. it doesn't affect the foreach whether the ienumerable is empty or not. only if it's null we shouldn't touch it